src/features/settings/SettingsPopover.tsx

```diff
@@ -1,41 +1,59 @@
-import { Dispatch, FC, RefObject } from "react"
-import { useClickOutside } from '@react-hookz/web';
-import { VscGithubInverted } from 'react-icons/vsc'
+import { Dispatch, FC, RefObject } from "react";
+import { useClickOutside } from "@react-hookz/web";
+import { VscGithubInverted } from "react-icons/vsc";
 import { TextWithLineAfter } from "../common/TextWithLineAfter";
-import { ThemeValue, themeLabels, themeValues, useThemeStore } from "../../store/themeStore";
-import { LanguageValue, languageLabels, languageValues, useLanguageStore } from "../../store/languageStore";
+import {
+  ThemeValue,
+  themeLabels,
+  themeValues,
+  useThemeStore,
+} from "../../store/themeStore";
+import {
+  LanguageValue,
+  languageLabels,
+  languageValues,
+  useLanguageStore,
+} from "../../store/languageStore";
 import { twJoin } from "tailwind-merge";
+import { useFormattedDate } from "./utils/useFormattedDate";
 
 interface SettingsPopoverProps {
-  settingsPopoverRef: RefObject<HTMLDivElement>
-  togglePopover: Dispatch<boolean>
+  settingsPopoverRef: RefObject<HTMLDivElement>;
+  togglePopover: Dispatch<boolean>;
 }
 
-const SettingsPopover: FC<SettingsPopoverProps> = ({ settingsPopoverRef, togglePopover }) => {
-  const { setTheme, theme } = useThemeStore()
-  const { language, setLanguage } = useLanguageStore()
+const SettingsPopover: FC<SettingsPopoverProps> = ({
+  settingsPopoverRef,
+  togglePopover,
+}) => {
+  const { setTheme, theme } = useThemeStore();
+  const { language, setLanguage } = useLanguageStore();
 
   const handleThemeChange = (newTheme: ThemeValue) => {
-    setTheme(newTheme)
-  }
+    setTheme(newTheme);
+  };
 
   const handleLanguageChange = (newLanguage: LanguageValue) => {
-    setLanguage(newLanguage)
-  }
+    setLanguage(newLanguage);
+  };
 
-  useClickOutside(
-    settingsPopoverRef,
-    () => togglePopover(false),
-    ['click', 'keydown']
-  )
+  useClickOutside(settingsPopoverRef, () => togglePopover(false), [
+    "click",
+    "keydown",
```

Comment:
remove `keydown` - remember to test closing the modal with spacebar when focused on the button

Reply:
removed